xml/admin_gui_dashboard.xml

```diff
@@ -170,7 +171,7 @@
      <term><guimenu>Block</guimenu></term>
      <listitem>
       <para>
-       View and manage block devices and their &iscsi; exports.
+       View detailed information and manage images, mirroring, and &iscsi;.
```

Comment:
Would it be prudent to use "RBD images" here, or would that be too technical?

Reply:
Considering it's an introduction, you could have either. At this point, it's just kept simple - whatever change you recommend.